Mathlib/Data/One/Lemmas.lean

```diff
@@ -0,0 +1,51 @@
+/-
+Copyright (c) 2023 Yael Dillies. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yael Dillies
+-/
+import Batteries.Tactic.Init
+import Mathlib.Data.One.Defs
+import Mathlib.Tactic.ToAdditive
+import Mathlib.Tactic.Lemma
+
+/-! # Lemmas about inequalities with `1`. -/
```

Comment:
This file looks identical to `Mathlib/Algebra/ZeroOne/Lemmas.lean`.

Reply:
Oops.